src/Raven.Server/ServerWide/ClusterStateMachine.cs

```diff
@@ -551,6 +585,12 @@ private void DeleteDatabaseRecord(TransactionOperationContext context, long inde
 
             // delete all values linked to database record - for subscription, etl etc.
             CleanupDatabaseRelatedValues(context, items, databaseName);
+
+            var transactionsCommands = context.Transaction.InnerTransaction.OpenTable(TransactionCommandsSchema, TransactionCommands);
+            using (Slice.From(context.Allocator, databaseName, out var loweredKey))
+            {
+                transactionsCommands.DeleteByKey(loweredKey);
```

Comment:
This is wrong. You are trying to delete where key = 'loweredKey`
But you can have more than one such command